core/ide/src/main/java/org/alice/ide/cascade/ExpressionCascadeManager.java

```diff
@@ -480,8 +470,19 @@ public void appendItems(List<CascadeBlankChild> items, BlankNode<Expression> bla
         }
       }
       if ((enumType != null) && this.areEnumConstantsDesired(enumType)) {
+        // default value- not all types have this method but we try. This approach only works for enums, since at this
+        // point in the code we only know what the desired type of the selection has to be, not what parameter we're
+        // attempting to fill.
+        Object defaultValue = null;
+        AbstractMethod mthd = type.getDeclaredMethod("getDefaultValue");
```

Comment:
Instead of this kind of reflection that only handles enums, it might be possible to change `ParamaterBlank` to not call `super.updateChildren()` and instead call a more specialized method on `ExpressionCascadeManager` like `appendItems` but with default in mind, or even in a parameter.
Let's merge this mostly as is, but maybe such an approach can get the next round of default values to show up?